src/grammar.lalrpop

```diff
@@ -96,9 +96,9 @@ ExternalFnArg: ExternalFnArg = {
 }
 
 StatementFn: UntypedStatement = {
-    <s:@L> <p:"pub"?> "fn" <n:VarName> "(" <a:Comma<FnArg>> ")" <return_annotation:("->" <Type>)?> "{" <b:Expr+> "}" <e:@L> => Statement::Fn {
+    <s:@L> <p:"pub"?> "fn" <n:VarName> "(" <a:Comma<FnArg>> ")" <return_annotation:("->" <Type>)?> <e:@L> "{" <b:Expr+> "}"  => Statement::Fn {
         doc: None,
-        location: location(s, e),
+        location: location(s, e - 1),
```

Comment:
Why is it - 1 ?

Reply:
It looks better (in my opinion)! One less char of underline.

`location(s, e) ` -> https://i.imgur.com/25oep7K.png
`location(s, e - 1) ` -> https://i.imgur.com/ffS3tf0.png

Reply:
What does it look like with a return annotation?

Reply:
Same thing, one extra char of whitespace without the - 1.

`location(s, e) ` -> https://i.imgur.com/N2oOabR.png
`location(s, e - 1) ` -> https://i.imgur.com/8recJ8u.png

Reply:
Nice! Thanks for sharing that extra info!

Reply:
No problem! I should try and come up with a better workflow for this than manually uploading screenshots to imgur.